src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -1,9 +1,60 @@
 package frc.robot.maps;
 
+import java.util.function.BooleanSupplier;
+
 import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+import com.chopshop166.chopshoplib.sensors.MockDigitalInput;
 
-// Need to get MAC address for roborio
 @RobotMapFor("Default")
 public class RobotMap {
 
+    public static class IntakeMap {
+        private final SmartMotorController deploymentMotor;
+        private final SmartMotorController rollerMotor;
+        private final BooleanSupplier insideLimit;
+        private final BooleanSupplier outsideLimit;
+
+        public IntakeMap() {
+            this.deploymentMotor = new SmartMotorController();
```

Comment:
You can call `this(new SmartMotorController(), ...)` instead of setting each item separately, if it's easier to read